api_draft.txt

```diff
@@ -0,0 +1,128 @@
+
+################################################################################
+Arimaa Game Server JSON API
+################################################################################
+
+#TODO - see todos below
+
+All requests and responses are in JSON.
+
+Shared types:
+<Message> = {"message":<String>}
+<Error> = {"error":<String>}
+<GameID> = <Int>   #Identifier for a game
+<Auth> = <String>  #Authorization token returned from logging in
+<GameAuth> = <String>  #Authorization token returned from joining a game
+<Username> = <String>
+<Timestamp> = <Float> #Seconds since the epoch (1970-01-01 00:00 UTC) (up to however much precision the server has)
+
+===ACCOUNTS========================================================
+To register an account:
+POST /accounts/register {"username":<Username>,"email":<String>,"password":<String>}
```

Comment:
Should we have a field for people to register bots? I don't like using only the username to distinguish bots.

What about country?